### PR TITLE
fix: add pattern for the manifest schema v2.0 [OREX-578]

### PR DESCRIPTION
**Pull request by genalt**
Added regex to validate the Application Identifier. Verified here https://www.jsonschemavalidator.net/

### CODE REVIEW

File: docs/schema/2.0/manifest.schema.json [hunk 1]
```diff
@@ -75,7 +75,9 @@
         },
         "identifier": {
           "type": "string",
-          "maxLength": 128
+          "minLength": 6,
+          "maxLength": 128,
+          "pattern": "^[a-zA-Z0-9_-]+$"
```
Comment (antonaudzei-outreach) [inline docs/schema/2.0/manifest.schema.json:80]: I think we should also allow dot char.
AppIds like 'com.example.myapp" are common in Apple and Google stores

**Review by antonaudzei-outreach — COMMENTED** [review]
Please update docs synchronously

File: docs/schema/2.0/manifest.schema.json (hunk 1 above)
Comment (genalt) [reply docs/schema/2.0/manifest.schema.json:80]: ok, makes sense

Comment (genalt) [reply docs/schema/2.0/manifest.schema.json]: added

**Review by nimal-outreach — APPROVED** [review]

**Pull request merged**